src/scales/scale.time.js

```diff
@@ -10,6 +10,10 @@ module.exports = function(Chart) {
 
 	var timeHelpers = helpers.time;
```

Comment:
I originally refactored out `helpers.time` so that it could be shared between the `time` and `timeseries` scales. Should we move them back to this file if we're not going to have two scales?